firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -399,44 +399,46 @@ void check_spoof_voltages( struct torque_spoof_t* spoof ) // L -> A, H -> B
     if ( abs( spoof_a_adc_volts - spoof_a_dac_current_volts ) >
             VOLTAGE_THRESHOLD )
     {
-        if ( current_ctrl_state.override_flag.voltage_spike_a == 0 )
+        if ( current_ctrl_state.override_flag.voltage_spike_a < 3 )
         {
-            current_ctrl_state.override_flag.voltage_spike_a = 1;
+            current_ctrl_state.override_flag.voltage_spike_a += 1;
         }
         else
         {
             DEBUG_PRINT( "* * ERROR!!  Voltage Discrepancy on Signal A. * *" );
 
             disable_control( );
             current_ctrl_state.override_flag.voltage = 1;
+            current_ctrl_state.override_flag.voltage_spike_a = 1;
         }
     }
     else
     {
         current_ctrl_state.override_flag.voltage = 0;
-        current_ctrl_state.override_flag.voltage_spike_a = 0;
+        current_ctrl_state.override_flag.voltage_spike_a = 1;
```

Comment:
I'm counting the spikes from 1 to 3.  At 3, the control is disabled.  This requires more testing.